# 2024-10-28T17:04:20.143Z:
test_suite_meta_data {
}
test_status: ERROR
platform_error {
  errors {
    summary {
      namespace {
        namespace: "com.google.testing.platform"
      }
      error_code: 2002
      error_name: "PLUGIN_ERROR"
      error_classification: "TEST"
      error_message: "Exception thrown during onAfterAll invocation of plugin com.android.tools.utp.plugins.host.apkinstaller.AndroidTestApkInstallerPlugin."
      stack_trace: "com.google.testing.platform.api.plugin.PluginException: Exception thrown during onAfterAll invocation of plugin com.android.tools.utp.plugins.host.apkinstaller.AndroidTestApkInstallerPlugin.\r\n\tat com.google.testing.platform.plugin.PluginLifecycleKt.invokeOrThrow(PluginLifecycle.kt:376)\r\n\tat com.google.testing.platform.plugin.PluginLifecycleKt.invokeOrThrow$default(PluginLifecycle.kt:367)\r\n\tat com.google.testing.platform.plugin.PluginLifecycle$onAfterAll$1.invoke(PluginLifecycle.kt:302)\r\n\tat com.google.testing.platform.plugin.PluginLifecycle$onAfterAll$1.invoke(PluginLifecycle.kt:292)\r\n\tat com.google.testing.platform.core.telemetry.common.noop.NoopDiagnosticsScope.recordEvent(NoopDiagnosticsScope.kt:35)\r\n\tat com.google.testing.platform.core.telemetry.TelemetryKt.recordEvent(Telemetry.kt:105)\r\n\tat com.google.testing.platform.core.telemetry.TelemetryKt.recordEvent$default(Telemetry.kt:98)\r\n\tat com.google.testing.platform.plugin.PluginLifecycle.onAfterAll(PluginLifecycle.kt:292)\r\n\tat com.google.testing.platform.executor.SingleDeviceExecutor$execute$3.invoke(SingleDeviceExecutor.kt:101)\r\n\tat com.google.testing.platform.executor.SingleDeviceExecutor$execute$3.invoke(SingleDeviceExecutor.kt:100)\r\n\tat com.google.testing.platform.result.TestResultListenerManager.afterTestSuite(TestResultListenerManager.kt:139)\r\n\tat com.google.testing.platform.executor.SingleDeviceExecutor.execute(SingleDeviceExecutor.kt:115)\r\n\tat com.google.testing.platform.RunnerImpl.run(RunnerImpl.kt:117)\r\n\tat com.google.testing.platform.server.strategy.NonInteractiveServerStrategy$run$5.invoke(NonInteractiveServerStrategy.kt:95)\r\n\tat com.google.testing.platform.server.strategy.NonInteractiveServerStrategy$run$5.invoke(NonInteractiveServerStrategy.kt:94)\r\n\tat com.google.testing.platform.core.telemetry.common.noop.NoopDiagnosticsScope.recordEvent(NoopDiagnosticsScope.kt:35)\r\n\tat com.google.testing.platform.core.telemetry.TelemetryKt.recordEvent(Telemetry.kt:66)\r\n\tat com.google.testing.platform.server.strategy.NonInteractiveServerStrategy.run(NonInteractiveServerStrategy.kt:94)\r\n\tat com.google.testing.platform.main.MainKt$main$4.invokeSuspend(Main.kt:75)\r\n\tat kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)\r\n\tat kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)\r\n\tat kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:279)\r\n\tat kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85)\r\n\tat kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59)\r\n\tat kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)\r\n\tat kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38)\r\n\tat kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source)\r\n\tat com.google.testing.platform.main.MainKt.main(Main.kt:73)\r\n\tat com.google.testing.platform.main.MainKt.main$default(Main.kt:35)\r\n\tat com.google.testing.platform.main.MainKt.main(Main.kt)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\r\n\tat com.google.testing.platform.launcher.Launcher.main(Launcher.java:149)\r\nCaused by: com.android.ddmlib.AdbCommandRejectedException: device \'emulator-5554\' not found\r\n\tat com.android.ddmlib.AdbHelper.setDevice(AdbHelper.java:1300)\r\n\tat com.android.ddmlib.AdbHelper.setDevice(AdbHelper.java:1319)\r\n\tat com.android.ddmlib.AdbHelper.executeRemoteCommand(AdbHelper.java:643)\r\n\tat com.android.ddmlib.AdbHelper.executeRemoteCommand(AdbHelper.java:511)\r\n\tat com.android.ddmlib.internal.DeviceImpl.executeShellCommand(DeviceImpl.java:739)\r\n\tat com.android.tools.utp.plugins.deviceprovider.ddmlib.DdmlibAndroidDevice.executeShellCommand(DdmlibAndroidDevice.kt)\r\n\tat com.android.tools.utp.plugins.deviceprovider.ddmlib.DdmlibAndroidDeviceController$executeAsync$deferred$1.invokeSuspend(DdmlibAndroidDeviceController.kt:165)\r\n\tat kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)\r\n\tat kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)\r\n\tat kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:571)\r\n\tat kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:750)\r\n\tat kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:678)\r\n\tat kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:665)\r\n"
    }
    cause {
      summary {
        error_message: "device \'emulator-5554\' not found"
        stack_trace: "com.android.ddmlib.AdbCommandRejectedException: device \'emulator-5554\' not found\r\n\tat com.android.ddmlib.AdbHelper.setDevice(AdbHelper.java:1300)\r\n\tat com.android.ddmlib.AdbHelper.setDevice(AdbHelper.java:1319)\r\n\tat com.android.ddmlib.AdbHelper.executeRemoteCommand(AdbHelper.java:643)\r\n\tat com.android.ddmlib.AdbHelper.executeRemoteCommand(AdbHelper.java:511)\r\n\tat com.android.ddmlib.internal.DeviceImpl.executeShellCommand(DeviceImpl.java:739)\r\n\tat com.android.tools.utp.plugins.deviceprovider.ddmlib.DdmlibAndroidDevice.executeShellCommand(DdmlibAndroidDevice.kt)\r\n\tat com.android.tools.utp.plugins.deviceprovider.ddmlib.DdmlibAndroidDeviceController$executeAsync$deferred$1.invokeSuspend(DdmlibAndroidDeviceController.kt:165)\r\n\tat kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)\r\n\tat kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)\r\n\tat kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:571)\r\n\tat kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:750)\r\n\tat kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:678)\r\n\tat kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:665)\r\n"
      }
    }
  }
}
issue {
  namespace {
    namespace: "com.google.testing.platform.runtime.android.driver.AndroidInstrumentationDriver"
  }
  severity: SEVERE
  code: 1
  name: "INSTRUMENTATION_FAILED"
  message: "Test run failed to complete. No test results"
}
